### ISSUE TITLE
Feature Request: Disable for unassigned and disabled characters

### ISSUE THREAD

**Issue by cregkly** [TeaWithLucas/DNDBeyond-DM-Screen#8]
This is great improvement on the original.

It would be nice if it didn't have to spent time scrapping the data for the characters that aren't used. Could we have some toggle controls to be able to turn them off?

**Comment by TeaWithLucas** [comment]
@cregkly  Yeah, that is something I want to update, it shouldn't take too long to implement
